core/metrics/src/pusher.rs

```diff
@@ -54,12 +54,36 @@ pub async fn push_forever(host_url: &str) {
 }
 
 pub async fn push(client: &Client, push_url: String) {
-    let current_metrics = crate::service::export_metrics().await;
+    let metrics = crate::service::export_metrics().await;
     let res = client
         .put(push_url.as_str())
-        .send_body(current_metrics)
+        .send_body(metrics.clone())
         .await;
-    log::trace!("Pushed current metrics {:#?}", res);
+    match res {
+        Ok(r) if r.status().is_success() => {
+            log::trace!("Metrics pushed: {}", r.status())
+        }
+        Ok(r) if r.status().is_server_error() => {
+            log::debug!("Metrics server error: {:#?}", r);
```

Comment:
In case of server errror just use `debug`. It is our (Golem Factory) fault, so nothin that should bother users.